js/utilities.js

```diff
@@ -1027,7 +1027,11 @@
             case 503:
                 return new module.ServiceUnavailableError(response.statusText, response.data);
             default:
-                return new Error(response.statusText, response.data);
+                if (response.statusText || response.data) {
```

Comment:
Was there an error response that didn't have statusText and data in it?

Reply:
In some cases `_responseToError` is being called as part of the `.catch()` callback for promises. It's used correctly there when an http request errors out and returns. But if there is an error in code execution, the `.catch()` will catch that and then call `_responseToError` with no data in it (sometimes it's `response.message`).